src/components/molecules/Alert/Alert.tsx

```diff
@@ -56,31 +40,63 @@ export const Alert = ({
         }
     };
 
-    return isVisible ? (
+    const alertIcon = alertIconMapping[severity];
+
+    if (!isVisible) return null;
+
+    return (
         <section
             aria-label={severity}
             role='alert'
-            className={clsx('rounded-xl', {
-                'border-2 border-yellow bg-yellow/20': variant === 'outlined', // should be changed to handle different colors
-                'bg-[rgba(41, 45, 63, 0.60)] border border-white-10 shadow-tab':
-                    variant === 'solid',
-            })}
+            className={clsx(
+                'flex w-full flex-row items-start justify-between gap-1 rounded-md border px-2.5 text-2xs text-neutral-50 laptop:text-xs',
+                severityStyle[severity],
+                {
+                    'py-1.5': !subtitle,
+                    'py-2': subtitle,
+                }
+            )}
         >
-            <div
-                className={clsx(
-                    'flex w-full flex-row items-center justify-between gap-1 rounded-xl px-5 py-3',
-                    {
-                        'bg-gradient-to-b from-[rgba(111,116,176,0.35)] to-[rgba(57,77,174,0)]':
-                            variant === 'solid',
-                    }
+            <div className='flex items-start gap-1.5 pr-4 laptop:gap-2'>
+                {alertIcon && (
+                    <span>
+                        {cloneElement(alertIcon, {
+                            className: clsx(
+                                alertIcon.props.className,
+                                iconClass,
+                                'mt-[4px] laptop:mt-0.5'
+                            ),
+                        })}
+                    </span>
                 )}
-            >
-                <div className='flex items-center gap-3'>
-                    <span>{alertIcon}</span>
-                    {children}
+                <div className='flex flex-col'>
+                    {title && (
+                        <h2
+                            className={clsx('text-xs leading-5', {
+                                'laptop:text-sm laptop:leading-[22px]':
+                                    subtitle,
+                            })}
+                        >
+                            {title}
+                        </h2>
+                    )}
+                    {subtitle && (
+                        <div className='leading-4 laptop:leading-5'>
+                            {subtitle}
+                        </div>
+                    )}
                 </div>
-                {showCloseButton && <CloseButton onClick={handleClose} />}
             </div>
+            <button
+                onClick={handleClose}
+                className={clsx(
+                    iconClass,
+                    'mt-[3px] flex-shrink-0 laptop:mt-0.5',
```

Comment:
again I think the top margin is also dependent on if subtitle is present or not.
When present, it should be 0

Reply:
Good call.

At first I didn't feel too comfortable with the close button being on a different vertical position from Alert Icon and title, but the designs are made to be that way, so I've followed your suggestion

Reply:
But top margin for Alert Icon is the same regardless of subtitle, so we can give off the effect of:

1) If title is kept in one line, Alert Icon is centered vertically 
2) If title has multiple lines OR subtitle is present, alert icon is aligned top